velox/expression/CastExpr-inl.h

```diff
@@ -126,6 +126,23 @@ StringView convertToStringView(
   return StringView(startPosition, writePosition - startPosition);
 }
 
+inline bool hasWhitespace(const StringView& value) {
```

Comment:
nit: perhaps, rename for clarity: hasLeadingOrTrailingWhitespace

Reply:
Done